MediaPlayer/MediaPlayer.module.less

```diff
@@ -0,0 +1,7 @@
+@import "../styles/skin.less";
+
+.mediaPlayer {
+	.applySkins({
+		height: 132px;
```

Comment:
Can we have this as a variable? like @agate-mediaplayer-height

Reply:
Sure, variable is added.